src/node/index.ts

```diff
@@ -105,7 +105,21 @@ export default class Node {
           plugins: [decompressTargz()],
         }).then(() => {
           fs.unlinkSync(downloadPath)
+          this.window.webContents.send('pointNode:finishDownload', true)
           resolve(this.installationLogger.log('Files decompressed'))
+
+          // stringify JSON Object
+          const jsonData = '{"nodeVersionInstalled":"' + version + '"}'
+          const  jsonContent = JSON.parse(jsonData)
+          const jsonParse = JSON.stringify(jsonContent)
```

Comment:
This is super weird. Instead of lines 112-114 please simply do:
```
fs.writeFile(path.join(pointPath, 'infoNode.json'), JSON.stringify({nodeVersionInstalled: version}), 'utf8', ...
```
Also you why do you use structure like this `{nodeVersionInstalled: version}` here while you are using structure like this `{version: 'old'}` [here](https://github.com/pointnetwork/pointnetwork-dashboard/pull/169/files#diff-3c294105e2f47172ca7029b0e00aeeae480029cc03981b4f2a0121d091cd13fdR120).

Shouldn't it be the same structure? Or do I miss something?